### PR TITLE
Updates for weasel

### PR DESCRIPTION
**Pull request by adrianeboyd**
## Description

Updates for weasel.

### Types of change

?

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] I ran the `update` scripts in the `.github` folder, and all the configs and docs are up-to-date.

### CODE REVIEW

**Review by svlandeg — APPROVED** [review]
Thanks so much for updating all of these! Only had a small nitpick question.

And yea, I can see how the CI chokes on wanting to run all of these projects :/ We also don't want this to be dozens of separate PRs, so I think we might just have to merge it being red (😳)

File: benchmarks/healthsea_spancat/README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+# 🪐 Weasel Project: Healthsea-Spancat
 
 This spaCy project uses the Healthsea dataset to compare the performance between the Spancat and NER architecture.
 
 ## 📋 project.yml
 
 The [`project.yml`](project.yml) defines the data assets required by the
 project, as well as the available commands and workflows. For details, see the
-[spaCy projects documentation](https://spacy.io/usage/projects).
+[Weasel documentation](https://github.com/explosion/weasel).
```
[the remark this answers is not in the document]
Comment (adrianeboyd) [reply benchmarks/healthsea_spancat/README.md:11]: This is just the auto `document` output from weasel. If we wanted to change this, we'd have to change it weasel.

[the remark this answers is not in the document]
Comment (adrianeboyd) [reply benchmarks/healthsea_spancat/README.md:11]: I think the more generic link is better especially if the docs move at some point.

[the remark this answers is not in the document]
Comment (svlandeg) [reply benchmarks/healthsea_spancat/README.md:11]: Ok!

**Pull request merged**